lib/stripe/application_fee.rb

```diff
@@ -16,5 +16,7 @@ def refund(params={}, opts={})
       # from the server
       self.refresh
     end
+    extend Gem::Deprecate
```

Comment:
Yeah, +1. I'd originally but the extends down near the usage so as to make it obvious to pull them out once we start to remove those deprecated methods. It is a little unusual though, so I've hoisted them in 437c2df.